composer.json

```diff
@@ -46,6 +47,7 @@
         "preferred-install": "dist"
     },
     "extra": {
+        "contao-manager-plugin": "Contao\\FaqBundle\\ContaoManager\\Plugin",
```

Comment:
Because we must retrieve that information from packagist to see if the package supports the manager.